ci/ray_ci/automation/filter_flaky_tests.py

```diff
@@ -1,29 +1,30 @@
 import sys
+import click
 
-from ci.ray_ci.utils import filter_out_flaky_tests
+from ci.ray_ci.utils import filter_flaky_tests
 from ray_release.configs.global_config import init_global_config
 from ray_release.bazel import bazel_runfile
 
 
-def main():
+@click.command()
+@click.argument("prefix", required=True)
```

Comment:
could you make this `prefix` an option rather than an argument?

Reply:
done